Classes/Seo/XmlSitemap/PagesXmlSitemapDataProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the "headless" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ *
+ * (c) 2020
+ */
+
+declare(strict_types=1);
+
+namespace FriendsOfTYPO3\Headless\Seo\XmlSitemap;
+
+class PagesXmlSitemapDataProvider extends \TYPO3\CMS\Seo\XmlSitemap\PagesXmlSitemapDataProvider
+{
+    protected function defineUrl(array $data): array
+    {
+        $typoLinkConfig = [
+            'parameter' => $data['uid'],
+            'forceAbsoluteUrl' => 0,
```

Comment:
why we need this?

Reply:
we have to override forceAbsoluteUrl (default in CORE is 1), we have to replace API url with frontendBase url